CMakeLists.txt

```diff
@@ -499,6 +502,7 @@ if(WIN32)
   list(
     APPEND 
       system_test_runner_sources
+        "source/tests/system/nimxlcterminaladaptor_restricted_driver_api_tests.cpp"
```

Comment:
The new test is [failing on our build](https://github.com/ni/grpc-device/runs/8253818570?check_suite_focus=true#step:4:870). We aren't blocking grpc-device PRs right now for failing System Tests since we have a few other intermittent but we should probably do one of two things for this new one.

We can either remove the new test file from the SystemTestsRunner along with any of the additions with `target_link_libraries` and `set_target_properties`.

Or we could try to update our windows grpcbot to pull in whatever feeds it needs to get these new tests to pass. The Windows feeds are defined [here](https://dev.azure.com/ni/DevCentral/_git/ni-central?path=%2Feng%2Fiac%2Fcfg_mgmt%2Fsalt%2Fsrv%2Fsalt%2Fgrpcbot%2Fwin%2Fresources%2Fdriver_feeds_win.yml&version=GBmain&_a=contents) in AzDo.

Reply:
And I'm fine with either update going in after this change, it's just something we'll need to address.

Reply:
I've created a task to address this next. Thanks for the detailed explanation!